swc/tests/versions_test.bzl

```diff
@@ -1,21 +0,0 @@
-"""Unit tests for starlark helpers
-See https://docs.bazel.build/versions/main/skylark/testing.html#for-testing-starlark-utilities
-"""
-
-load("@bazel_skylib//lib:unittest.bzl", "asserts", "unittest")
-load("//swc/private:versions.bzl", "TOOL_VERSIONS")
-
-def _smoke_test_impl(ctx):
-    env = unittest.begin(ctx)
-
-    # TODO: assert instead that the TOOL_VERSIONS.keys()[0] is the newest version
-    # and version numbers down from there are descending
-    asserts.equals(env, "v1.3.42", TOOL_VERSIONS.keys()[0])
-    return unittest.end(env)
-
-# The unittest library requires that we export the test cases as named test rules,
-# but their names are arbitrary and don't appear anywhere.
-t0_test = unittest.make(_smoke_test_impl)
-
-def versions_test_suite(name):
```

Comment:
We don't want to keep any of these?

Reply:
There is one test and it doesn't assert anything. It would be useful to write some starlark tests but if we didn't yet, we probably won't